raiden/ui/runners.py

```diff
@@ -219,19 +227,14 @@ def sig_set(sig=None, _frame=None):
         finally:
             self._shutdown_hook()
 
-            app_.stop()
+            for task in gevent_tasks:
+                task.kill()
 
-            def stop_task(task):
-                try:
-                    if isinstance(task, Runnable):
-                        task.stop()
-                    else:
-                        task.kill()
-                finally:
-                    task.get()  # re-raise
+            for task in runnable_tasks:
+                task.stop()
 
             gevent.joinall(
-                set(gevent.spawn(stop_task, task) for task in tasks),
+                set(gevent_tasks + runnable_tasks),
```

Comment:
AFAICS this will no longer re-raise exceptions (since the `task.get()` was removed). Is that intentional?